pkg/scheduler/api/resource_info.go

```diff
@@ -94,7 +94,7 @@ func NewResource(rl v1.ResourceList) *Resource {
 
 // IsEmpty returns bool after checking any of resource is less than min possible value
 func (r *Resource) IsEmpty() bool {
-	if !(r.MilliCPU < minMilliCPU && r.Memory < minMemory) {
+	if r.MilliCPU >= minMilliCPU || r.Memory >= minMemory {
 		return false
```

Comment:
The minMilliCPU and mini minMemory is float64 type. Do not use equal to compare.

Reply:
Thanks for your advice. I have two ideas to deal with the problem. PTAL @william-wang , thanks a lot.

The first:
``` go
func (r *Resource) IsEmpty() bool {
	// Do not use equal to compare floats.
	const precision = 1e-6
	if r.MilliCPU > (minMilliCPU-precision) || r.Memory > (minMemory-precision) {
		return false
	}

	for _, rQuant := range r.ScalarResources {
		if rQuant >= minMilliScalarResources {
			return false
		}
	}

	return true
}
```
The second:
``` go
func (r *Resource) IsEmpty() bool {
	// Do not use equal to compare floats. Compare if a is greater than or equal to b
	geFn := func(a, b float64) bool {
		const precision = 1e-6
		return a > (b - precision)
	}
	if geFn(r.MilliCPU, minMilliCPU) || geFn(r.Memory, minMemory) {
		return false
	}

	for _, rQuant := range r.ScalarResources {
		if rQuant >= minMilliScalarResources {
			return false
		}
	}

	return true
}
```